pre_commit_hooks/safety_check.py

```diff
@@ -1,18 +1,38 @@
 from __future__ import print_function
+
+import argparse
 import sys
+
 from safety.cli import check
 
 
 def main(argv=None):
-    if argv is None:
-        argv = sys.argv[1:]
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--full-report",
+        default="--full-report",
+        const="--full-report",
+        action="store_const",
+    )
+    parser.add_argument(
+        "--short-report", dest="full_report", const="--short-report", action="store_const"
+    )
+    parser.add_argument("--ignore", action="append")
```

Comment:
Why specifying this ? Shouldn't it be passed through to `safety_check` in the `rest` variable ?
I do not want to re-define all existing / supported `safety` flags here

Reply:
this is because I want to support `--ignore=1,2,3`

Reply:
>  I do not want to re-define all existing / supported safety flags here

there's no need to redefine them all, only the ones needed to change the behaviour